### ISSUE TITLE
multiple domains?

### ISSUE THREAD

**Issue by ratacibernetica** [antoniomika/sish#88]
Is it possible to handle multiple domains with SSL in the same server?
Maybe loading the *.pem files by matching the domain requested by the client?

```
ssl/mydomain.com/ -> *.pem files
ssl/mydomain2.net/ -> *.pem files
```

**client 1:**

```
ssh -R foo:80:localhost:8080 mydomain.com
```

would give `foo.mydomain.com`

**client 2:**

```
ssh -R foo:80:localhost:8080 mydomain2.net
```

would give `foo.mydomain2.net`

I tried playing with the code but I'm an absolute noob in Go and couldn't figure out how to find out the client's requested domain.

**Comment by antoniomika** [comment]
Hey @ratacibernetica! 

The difficult thing here is that ssh doesn't send the hostname that you're connecting to (mydomain.com or mydomain2.net). In the v1.0 branch I've added support for custom domains, where you would provide the full domain as the subdomain parameter. That feature verifies DNS if the configured domain isn't suffixed and that maybe be an issue if using wild card certs.

**Comment by antoniomika** [comment]
@ratacibernetica I just added support for adding multiple ssl certs to sish and added the ability to bind multiple domains other than the main domain without requiring dns verification.

This is in the `v1.0-work` branch and I updated the readme for that. More or less, you add the certificates you want sish to load into the `ssl/` directory named `domain.crt` and `domain.key` for each. sish will load all of the certificates from there. To use it, set the `--bind-hosts` flag to a comma separated list of domains that can be used for provisioning subdomains.
